playground/cli/jsonToPDF/script.js

```diff
@@ -0,0 +1,58 @@
+const Mustache = require("mustache"); 
+const fs = require('fs')
+const createHTML = require('create-html')
+const htmlToPdf = require('html-to-pdf');
+
+main();
+function main(){   
+    let data;
```

Comment:
I believe you can use `const` for most of the variables here, but it isn't really important.